### DISCUSSION TITLE
Failed to update auth collection field

### DISCUSSION THREAD

**Discussion by Graciasjr** [pocketbase/pocketbase#4646] [Q&A]
Hello pb community. I have a little problem with updating data in auth type collections. 

Here is the update script: 

![pb script](https://github.com/pocketbase/pocketbase/assets/89590621/8e985bd4-e8ce-42bd-a042-cfae9fff393f)

This is what I get: 
![console print](https://github.com/pocketbase/pocketbase/assets/89590621/d093a34a-5e4a-4975-b823-d52a889ea8de)

I'm sure I've done something wrong and your help would be greatly appreciated. THANKS

**Comment by Graciasjr** [comment]
obviously I didn't give the necessary detail
 👇🏽this is the meta details in the pb_log
![pb log](https://github.com/pocketbase/pocketbase/assets/89590621/89a4f3d0-745a-4e20-932d-8cea240d0907)

**Reply by ganigeorgiev** [reply]
The above error just indicates that the user you are trying to update doesn't exist (or that you don't have permissions to perform the update operation).

To handle JS SDK errors please refer to [Error handling](https://github.com/pocketbase/js-sdk?tab=readme-ov-file#error-handling).

Also note that the error in your initial screenshot is not related to PocketBase.

**Comment by Graciasjr** [comment]
The user does exist, I even tried to make the query by copying the id directly from the database. So I assume the error must be related to permissions.
 Here is a screenshot of the API rules in the GUI: 
![api rules1](https://github.com/pocketbase/pocketbase/assets/89590621/92473354-9b33-4067-84dd-de4c7941d335)
![api rules2](https://github.com/pocketbase/pocketbase/assets/89590621/83dc30e7-eb0e-491c-a58c-722ab1e66435)
![api rules3](https://github.com/pocketbase/pocketbase/assets/89590621/1fde3c68-366c-4ec7-8c20-29ed24d49a34)


Did I do something wrong here?
regarding the errors, I take notes

**Reply by ganigeorgiev** [reply]
Make sure that you are sending the requested as authenticated user (aka. that `local.pb.authStore.token` is set and valid).

Also I'd recommend disabling the "Manage rule" unless that is what you want because the above practically will allow the user to update its own verification, email, etc. fields without the need of confirmation (make sure to check the note below the input field).

**Reply by Graciasjr** [reply]
Great, thanks .
I will try it.

**Reply by Graciasjr** [reply]
Otherwise, what to do when `locals.pb.authstore.token` and `locals.pb.authstore.valid` is not valid?
